app/services/streams/twitch_access_token/get.rb

```diff
@@ -7,11 +7,11 @@ def self.call
       end
 
       def call
-        token, exp = Rails.cache.fetch(ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY)
+        token, exp = RedisRailsCache.fetch(ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY)
 
         if token.nil? || Time.zone.now >= exp
           token, exp = get_new_token
```

Comment:
@benhalpern question, is this expiration constantly changing and that's why we always reset it?